test/foundry/fork/arbitrum/ArbitrumSettings.t.sol

```diff
@@ -11,7 +11,10 @@ contract ArbitrumSettings is NexusTest_Base {
     address public constant ENTRYPOINT_ADDRESS = 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789;
     address public constant MODULE_ADDRESS = 0x0000001c5b32F37F5beA87BDD5374eB2aC54eA8e;
     address public constant USDC_ADDRESS = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8;
-    string public constant DEFAULT_ARBITRUM_RPC_URL = "https://arbitrum-one-archive.allthatnode.com";
+    string public constant DEFAULT_ARBITRUM_RPC_URL = "https://public.stackup.sh/api/v1/node/arbitrum-one";
+    //string public constant DEFAULT_ARBITRUM_RPC_URL = "https://arb-pokt.nodies.app";
+    //string public constant DEFAULT_ARBITRUM_RPC_URL = "https://rpc.ankr.com/arbitrum";
+    //string public constant DEFAULT_ARBITRUM_RPC_URL = "https://arbitrum-one-rpc.publicnode.com";
```

Comment:
```suggestion
```
Did you add them on purpose?  or just forgot to remove ?

Reply:
yeah, I added them on purpose as backup RPC URLS
as for me some tests were not passing because url unavailable. so can easily switch between them